CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## Unreleased
+
+### Enhancements
```

Comment:
@harendra-kumar Can you comment / add the Changelog entry for the new plugin pass

Reply:
- Now forces inline on binders that return a type annotated with `Fuse`.